libparsec/crates/tests_fixtures/src/lib.rs

```diff
@@ -1,31 +1,22 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
 // Must rename `tmp_path` module to prevent its `tmp_path` item from being shadowed
-#[path = "./tmp_path.rs"]
+#[path = "tmp_path.rs"]
 mod tmp_path_mod;
 mod trustchain;
 
 pub use tmp_path_mod::*;
 pub use trustchain::*;
 
-// Reexport 3rd parties needed by `parsec_test` macro
-pub use env_logger;
-pub use rstest;
-#[cfg(not(target_arch = "wasm32"))]
-pub use tokio;
-// Pretty assertions are super useful in tests, the only issue is we cannot use star-use
-// to import them (given they shadow the default asserts), so we expose them with a
-// sightly different name
-pub use pretty_assertions::{
-    assert_eq as p_assert_eq, assert_matches as p_assert_matches, assert_ne as p_assert_ne,
-    assert_str_eq as p_assert_str_eq,
-};
+pub use libparsec_tests_lite::*;
 
 // Reexport so that `use libparsec_tests_fixtures::prelude::*` is the single-no-brainer-one-liner™
 pub use libparsec_testbed::*;
-pub use libparsec_tests_macros::parsec_test;
 pub use libparsec_types::fixtures::*;
 
 pub mod prelude {
     pub use super::*;
 }
+
+#[cfg(target_arch = "wasm32")]
+pub use wasm_bindgen_test;
```

Comment:
Isn't it already provided by `libparsec_tests_lite` ?

Reply:
done